### PR TITLE
adding calculator.js with some calculator functions

### PR DESCRIPTION
**Pull request by rburlevich**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,16 @@
 	<head>
 	</head>
 	<body>
-	Hello!
+		Hello!
```
Comment (NikitaSedyx) [inline index.html]: 1. new folder for task
2. wrong indentations, 2 spaces is correct
3. empty line at the EOF in each file, you see red sign

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]

File: calculator/js/calculator.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function addition(a, b) {
+return a + b;
```
Comment (NikitaSedyx) [inline calculator/js/calculator.js]: nit: could you make indentations?

**Review by NikitaSedyx — APPROVED** [review]
please fix code style

File: calculator/js/calculator.js (hunk 2 above)
Comment (rburlevich) [reply calculator/js/calculator.js]: done, please review again

Comment (NikitaSedyx) [reply calculator/js/calculator.js]: you have my approve, it was a nit

**Pull request closed without merging**